### ISSUE TITLE
Ajax Upload Not Working

### ISSUE THREAD

**Issue by nickff** [Bruno17/MIGX#277]
Hi Bruno,

So I recently realized that two sites on one of my servers are having issues with the Ajax upload feature all of a sudden. Everything is working fine other than the multi-upload, which was working not too long ago. 

The folder is correctly created for each resource with the proper permissions, but when I open the multi-upload box there is no "upload files" button and I can't drag images into it (screenshot attached). I tried updating MIGX and MODX to the latest respective versions, but no luck.

![blank-ajax-upload](https://cloud.githubusercontent.com/assets/386558/23222994/b85fd070-f8f7-11e6-8477-d68b4a4eceb3.png)

Any ideas would be much appreciated!

**Comment by nickff** [comment]
Also, here's a view of the console error being generated:

![screen shot 2017-02-23 at 10 52 49 am](https://cloud.githubusercontent.com/assets/386558/23266611/42aed9c4-f9b6-11e6-9c11-0f3d25f1e6a3.png)

**Comment by nickff** [comment]
So I've figured out that this is due to having installed an SSL certificate on the website. The call to http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js is not secure, so it's not loading.

The only reference I can find to this file is in "ajaxupload.html" which is in the core/packages/migx-2.9.6-pl/ folder. But when I update that to http[b]s[/b]://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js it's still not working and the change isn't reflecting (even after clearing core/cache and my browser cache etc).

Any thoughts on how to update that path to be secure would be much appreciated.

**Comment by Bruno17** [comment]
I think the file is this one https://github.com/Bruno17/MIGX/blob/master/core/components/migx/templates/mgr/iframechunks/ajaxupload.html

**Comment by jpdevries** [comment]
Instead of requesting jquery over http you can use a relative protocol like `//code.jquery.cdn`

**Comment by nickff** [comment]
Hi guys, thanks for the info.

Bruno, that's the file I updated and it's not making a difference. Any extra steps needed to make that change take effect other than edit then upload?

**Comment by nickff** [comment]
@Bruno17 You're right - I was in core/packages/ rather than core/components/. Updating the path to jquery in that file fixed it.

Perhaps this is something you can implement in future versions, so manual editing is not necessary for HTTPS sites?

**Comment by Bruno17** [comment]
@nickff Did you change it to a relative protocol, as suggested by jpdevries?
which did work for you? 
Will change it in future versions, if I know its working for all protocols

**Comment by nickff** [comment]
I used the https protocol for the jquery path.
